app-script-payslip-sender/Payslip.js

```diff
@@ -0,0 +1,21 @@
+function PayslipSender() {
+  var spreadsheet = SpreadsheetApp.getActive().getSheetByName('Sheet1');
+  var dataRange = spreadsheet.getRange("A2:D2").getValues();
+  Logger.log(dataRange);
+  for(var i=0; i < dataRange.length; i++) {
+    var employeeData = dataRange[i];
+    var employeeName = employeeData[1];
+    var salary = employeeData[3];
+    var email = employeeData[2];
+    var PayslipMessageContent = PayslipMessage(employeeName, salary);
+    Logger.log(PayslipMessageContent);
+    MailApp.sendEmail(email, 'Payslip', PayslipMessageContent)
+  }
+}
+function PayslipMessage(employeeName, salary) {
+  var message = "Hi " + employeeName + "\n";
+  message += "Your salary for the month of may has been deposited!\n";
```

Comment:
can you find better way to generate a string/paragraph with interpolation?

Reply:
Used template literals to embed expressions within backticks and placeholders for placing expressions directly within strings.